packages/block-library/src/media-text/media-container.native.js

```diff
@@ -1,51 +1,85 @@
 /**
  * External dependencies
  */
-import { View, Image, ImageBackground } from 'react-native';
+import { Dimensions, View, ImageBackground, Text, TouchableWithoutFeedback, Platform } from 'react-native';
+import {
+	requestMediaImport,
+	mediaUploadSync,
+	requestImageFailedRetryDialog,
+	requestImageUploadCancelDialog,
+} from 'react-native-gutenberg-bridge';
 
 /**
  * WordPress dependencies
  */
-import { IconButton, Toolbar, withNotices } from '@wordpress/components';
+import {
+	Icon,
+	IconButton,
+	Toolbar,
+	withNotices,
+} from '@wordpress/components';
 import {
 	BlockControls,
 	BlockIcon,
-	MediaPlaceholder,
 	MEDIA_TYPE_IMAGE,
+	MEDIA_TYPE_VIDEO,
+	MediaPlaceholder,
 	MediaUpload,
+	MediaUploadProgress,
+	VideoPlayer,
 } from '@wordpress/block-editor';
 import { Component } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { isURL } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
+import styles from './style.scss';
 import icon from './media-container-icon';
+import SvgIconRetry from './icon-retry';
 
-export function calculatePreferedImageSize( image, container ) {
-	const maxWidth = container.clientWidth;
-	const exceedMaxWidth = image.width > maxWidth;
-	const ratio = image.height / image.width;
-	const width = exceedMaxWidth ? maxWidth : image.width;
-	const height = exceedMaxWidth ? maxWidth * ratio : image.height;
-	return { width, height };
-}
+/**
+ * Constants
+ */
+// For Android it will only work with the first element of the array (images)
+const ALLOWED_MEDIA_TYPES = [ MEDIA_TYPE_IMAGE, ...( Platform.OS === 'ios' ? [ MEDIA_TYPE_VIDEO ] : [] ) ];
```

Comment:
I'd suggest [Platform.select](https://facebook.github.io/react-native/docs/platform-specific-code#platform-module) for a bit more clarity:

```suggestion
const ALLOWED_MEDIA_TYPES = Platform.select({
	android: [ MEDIA_TYPE_IMAGE ],
	default: [ MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO ],
});
```

Reply:
I agree, way better to see! I'll change it